addon/components/ember-tbody/component.js

```diff
@@ -260,6 +261,12 @@ export default Component.extend({
     );
   },
 
+  _updateColumnTree: observer('wrappedRows.[]', function() {
+    next(() => {
```

Comment:
This timer is flaky. We need to wait for all table rows to be rendered but sometimes rows aren't rendered yet.

Reply:
Using `scheduler.schedule('affect'` & `Ember.run.next` is the only solution found to wait for rendered rows.